src/content/docs/telemetry-data-platform/ingest-apis/introduction-event-api.mdx

```diff
@@ -66,36 +66,23 @@ The Event API is an asynchronous endpoint. This allows you to send a very large
 
 To send a custom event to your New Relic account:
 
-1. [Register](#register) an Insert API key.
-2. Before creating custom events or attributes, review New Relic's list of [reserved terms used by NRQL](https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/insights-custom-data-requirements-limits#reserved-words).
+1. You'll use a New Relic account [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key).
```

Comment:
Do we want to get more specific and tell them to go get their account license key?